src/stories/input/Input.module.scss

```diff
@@ -99,42 +99,54 @@
             display: none;
         }
 
-        &[data-status="disabled"] {
-            color: var(--pte-colors-contentDisabled);
+        &[data-disabled=true] {
```

Comment:
same note as prior—status styles override disabled so curious why we are separating these out from a style perspective

Reply:
Yes reverting back to `data-status="disabled"` across all the input types